x-pack/test/functional/es_archives/cases/default/mappings.json

```diff
@@ -298,13 +298,8 @@
           }
         },
         "type": {
-          "fields": {
-            "keyword": {
-              "ignore_above": 256,
-              "type": "keyword"
-            }
-          },
-          "type": "text"
```

Comment:
@stephmilovic is there a strong reason the `type` field was mapped as `string + keyword` here?
If we define it as `keyword` as we enforce with all new saved objects' indices, then we are able to perform aggregated searches over that field.

In fact, I perform one in this very same PR, and the mapping above is causing functional tests to fail with the following error:

> Reason: Text fields are not optimised for operations that require per-document field data like aggregations and sorting, so these operations are disabled by default. Please use a keyword field instead. Alternatively, set fielddata=true on [type] in order to load field data by uninverting the inverted index. Note that this can use significant memory.